resources/fixtures/Showlite-Flood-Light-Panel-144x10mm-LED-RGBW.qxf

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE FixtureDefinition>
+<FixtureDefinition xmlns="http://www.qlcplus.org/FixtureDefinition">
+ <Creator>
+  <Name>Q Light Controller Plus</Name>
+  <Version>4.10.3</Version>
+  <Author>Ex</Author>
+ </Creator>
+ <Manufacturer>Showlite</Manufacturer>
+ <Model>Flood Light Panel 144x10mm LED RGBW</Model>
+ <Type>Color Changer</Type>
+ <Channel Name="Dimmer">
+  <Group Byte="0">Intensity</Group>
+ </Channel>
+ <Channel Name="Red">
+  <Group Byte="0">Intensity</Group>
+  <Colour>Red</Colour>
+ </Channel>
+ <Channel Name="Green">
+  <Group Byte="0">Intensity</Group>
+  <Colour>Green</Colour>
+ </Channel>
+ <Channel Name="Blue">
+  <Group Byte="0">Intensity</Group>
+  <Colour>Blue</Colour>
+ </Channel>
+ <Channel Name="White">
+  <Group Byte="0">Intensity</Group>
+  <Colour>White</Colour>
+ </Channel>
+ <Channel Name="Speed">
+  <Group Byte="0">Speed</Group>
+ </Channel>
+ <Channel Name="Function">
+  <Group Byte="0">Effect</Group>
+  <Capability Min="0" Max="15">no function</Capability>
+  <Capability Min="16" Max="31">sence R-G-B-RG-G</Capability>
```

Comment:
I don't think "sence" is an existing english word.
I know it's in the manual, but it's wrong.
They probably meant "sequence" ?